src/hooks/useKeyboardControls.ts

```diff
@@ -12,6 +12,12 @@ export const MovementKeys = {
   right: ["right", "d"],
 };
 
+export const ShortcutKeys = {
+  chat: ["c"],
+  schedule: ["s"],
+  find: ["f"],
+};
```

Comment:
Instead of adding these to `src/hooks/useKeyboardControls.ts`, it might make sense to move both  `MovementKeys` and `ShortcutKeys` to somewhere like `src/settings`.

They could probably also be combined into the same `const` so that we have a single 'source of truth' for all our shortcuts, something like:

```typescript
export const KeyboardShortcutKeys = {
  chat: "c",
  find: "f",
  movement: {
    up: ["up", "w"],
    down: ["down", "s"],
    left: ["left", "a"],
    right: ["right", "d"],
  },
  schedule: "s",
}
```

It would probably also be good to rename `src/hooks/useKeyboardControls.ts` to `src/hooks/useMapKeyboardControls.ts` or similar to differentiate it (including updating the props/hook to match the filename), since we're adding more shortcuts to the platform now.

Reply:
For these minimal changes, yeah, more than happy for you to tweak the existing code for it.

Reply:
done, but made it KEYBOARD_SHORTCUTS (to stay inline with upper casing in `settings.ts`) and placed the last in the file since could not detect any order to follow. also renamed that adding `Map` into `useMapKeyboardControl*`

Reply:
While most things follow THIS_PATTERN, we have a few examples in `src/settings` already that don't. I feel like this is a case that make more sense to be `KeyboardShortcutKeys` or similar rather than `KEYBOARD_SHORTCUTS`, as we are using it sort of like an `enum`.

Reply:
You are the boss, I will change (I have seen only few of such in there).

Reply:
> I have seen only few of such in there

Yeah, there's not many in there that break THIS_PATTERN currently. That file (and it's code style) is a fair bit of legacy code/new stuff hacked into it rather than being overly 'planned'/thought through. So That pattern may change in future (eg. in `utils/*` and `types/*` we export `const`s/`enum`s/etc all the time that don't follow THAT_PATTERN; so it's established in the codebase in general, just not so much represented in `src/settings` currently)

Reply:
> You are the boss

Ha! I'm just a guy with strong opinions on our codebase 😛 

> I will change 

Sounds good, thanks :)

Reply:
done on that rename in 27dae04ab854b2d5429f261bc0fca332b2bd1a54 ATM.  I will leave one more question in a dedicated thread.

Reply:
Sounds good